packages/core/services/managers/layer-manager.ts

```diff
@@ -2,26 +2,29 @@ import { Injectable } from '@angular/core';
 import { AgmBicyclingLayer } from '../../directives/bicycling-layer';
 import { AgmTransitLayer } from '../../directives/transit-layer';
 import { GoogleMapsAPIWrapper } from '../google-maps-api-wrapper';
-import { BicyclingLayer, BicyclingLayerOptions, GoogleMap, TransitLayer, TransitLayerOptions } from '../google-maps-types';
+
+interface IVisibility {
```

Comment:
why recreate the IVisibility interface? not better to import from google-maps-api-wrapper?

Reply:
It's not exported and I don't want it to be. As mentioned above this interface should go away, but this is where it would actually be a breaking change.